### ISSUE TITLE
[Artist CTA] Escape key should dismiss modal.

### ISSUE THREAD

**Issue by alloy** [artsy/force#841]
For accessibility purposes and cases where (due to e.g. badly calibrated monitors) the dismiss X is not visible, pressing the ESC key should dismiss the modal.
